am-core-web-service/src/main/resources/db/migration/V6__classroom.sql

```diff
@@ -0,0 +1,18 @@
+--crear la tabla academic_period
```

Comment:
Please change sql name to "V7__classroom_table.sql" and add some classrooms for area in dev env